tests/format_utils/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023 The Khronos Group Inc.
+# Copyright 2023 Valve Corporation
+# Copyright 2023 LunarG, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+find_package(GTest REQUIRED CONFIG)
+find_package(magic_enum REQUIRED CONFIG)
+
+include(GoogleTest)
```

Comment:
Note I don't like how this code is repeated in our CMake testing code. I'll make a fix for this in another PR though. It's currently consistent as is.